command/harness/setup.go

```diff
@@ -215,7 +215,6 @@ func HandleSetup(
 		WithField("ip", instance.Address).
 		WithField("id", instance.ID).
 		WithField("instance_name", instance.Name).
-		WithField("response", fmt.Sprintf("%+v", resp)).
```

Comment:
can you please point me to the pr where this was added?

Reply:
Not sure. I haven't added it, it must be there from a long time

Reply:
why not just revert the change that was printing the extra info

Reply:
The log line to log the response was there from a long time I guess. Earlier the response had only instance id and IP. Now it has a lot more things required for CDE

We still need those information to be returned for CDE, including the certs and keys.
Just that we don't have to log them

Reply:
Since we already log instance id and IP above it
```
		WithField("ip", instance.Address).
		WithField("id", instance.ID).
```
I have removed the response part as it would have redundant info + info required for CDE